build/media_source/mod_community_info/css/community-info.css

```
@@ -0,0 +1,39 @@
+.select-location {
+  padding-left: 1em;
+  margin-bottom: .5rem;
+  font-weight: 400;
+}
+.text-right {
+  text-align: end;
+}
+.module-wrapper .card-header {
+  flex-wrap: wrap;
+}
+.intro-txt {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  justify-content: space-between;
+  margin-bottom: 1rem;
+}
+.info-block {
+  margin-bottom: 1rem;
+}
+.info-block img {
+  max-width: 100px;
+  margin: 1rem 0 0 .5rem;
+}
+.info-block a.no-link::before {
+  padding: 0;
+  content: "";
+}
+.mod-community-info h3 {
+  font-size: 1.05rem;
+}
+.mod-community-info.modal .btn-primary {
+  color: var(--btn-primary-color) !important;
+  background-color: var(--btn-primary-bg) !important;
+}
+.mod-community-info .float-right {
+  float: right;
```

please use logical css properties

```suggestion
  float: inline-end;
```

---

Done, thanks.